src/rust/ensogl/src/display/render/composer.rs

```diff
@@ -109,7 +109,8 @@ impl ComposerPass {
             let name    = format!("pass_{}",output.name());
             let args    = (self.width,self.height);
             let texture = uniform::get_or_add_gpu_texture_dyn
-                (&self.context,&self.variables,&name,output.internal_format,output.item_type,args);
+                (&self.context,&self.variables,&name,output.internal_format,output.item_type,args,
+                 Some(output.parameters));
```

Comment:
It took me a while to understand what a field "output pass . parameters" mean. It should be named `texture_parameters` instead.